logging/interface.go

```diff
@@ -0,0 +1,43 @@
+// Package logging defines a logging interface for quic-go.
+// This package should not be considered stable
+package logging
+
+import (
+	"net"
+	"time"
+
+	"github.com/lucas-clemente/quic-go/internal/congestion"
+	"github.com/lucas-clemente/quic-go/internal/protocol"
+	"github.com/lucas-clemente/quic-go/internal/wire"
+)
+
+type Tracer interface {
```

Comment:
Any idea for a better name?

Reply:
Not really. Logger maybe, but that's already taken.